jest.config.js

```diff
@@ -1,15 +1,8 @@
-import('jsdom');
-import 'jsdom-global/register.js';
 
 export default async () => {
-  const SlimModule = await import('./dist/index.js');
-  global.Slim = SlimModule.Slim;
   return {
     testPathIgnorePatterns: ['/old-src/'],
     testURL: 'http://localhost/',
-    testEnvironment: 'jsdom',
-    globals: {
-      window: {},
-    },
+    testEnvironment: 'jsdom'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/slimjs/slim.js/pullRequest?prid=8391259)
```suggestion
    testEnvironment: "jsdom"
```